pkg/repositories/mongo/league/delete.go

```diff
@@ -0,0 +1,40 @@
+package league
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+
+	"github.com/jairogloz/go-l/pkg/domain"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+var (
+	ErrDeleteLeague = errors.New("error deleting league")
+)
+
+// Used for delete one league by id from the database
```

Comment:
Debe de iniciar con el nombre de la función, en este caso "Delete user for ........"